conf/application.conf

```diff
@@ -47,7 +48,7 @@ microservice {
 
     si-protected-user-list-admin {
```

Comment:
The port for si-protected-user-list-admin is currently set to 8507 in the service manager configuration. The frontend uses 8508. Ah, I see 8182 is the silent login admin backend. I'd recommend using the new values so service manager can be used with either service until silent login is decommissioned.